app.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics.controller");
+
+app.use(express.json());
+
+app.get(`/api/topics`, getTopics);
+
+app.use((err, req, res, next) => {
+  console.error(err);
+
+  if (err.name === "SyntaxError") {
+    return res.status(400).json({ error: "Invalid JSON" });
+  } else if (err.name === "UnauthorizedError") {
+    return res.status(401).json({ error: "Unauthorized" });
+  }
+  res.status(500).json({ error: "Internal Server Error" });
+});
+app.listen(9090, (error) => {
+  if (error) {
+    console.log(error);
+  } else {
+    console.log("server is listening on port 9090");
+  }
+});
```

Comment:
This doesn't want to be in the app.js - right now we're only connecting to the server through the testing (with supertest), not making our own requests with an external app like insomnia.

We can move this to the listen.js for now - that means its out of the way and wont interfere with our tests